react/components/RenderProvider.tsx

```diff
@@ -120,6 +122,7 @@ class RenderProvider extends Component<Props, RenderProviderState> {
     if (!production) {
       emitter.addListener('localesUpdated', this.onLocalesUpdated)
       emitter.addListener('extensionsUpdated', this.updateRuntime)
+      document.addEventListener('visibilitychange', this.onVisibilityChanged)
```

Comment:
I would prefer if we don't change the `RenderProvider` and instead do something like this inside the `if` that checks `emittersByWorkspace[`${account}/${workspace}`]`

```  // Share SSE connections for same account and workspace
  if (!emittersByWorkspace[`${account}/${workspace}`]) {
    emittersByWorkspace[`${account}/${workspace}`] = []
    emittersByWorkspace[`${account}/${workspace}`].eventSource = initSSE(account, workspace, baseURI)
    if (!production ) {
      document.addEventListener('visibilitychange', () => {
        if (!document.hidden && emittersByWorkspace[`${account}/${workspace}`].eventSource.readystate == ruim) {
          emittersByWorkspace[`${account}/${workspace}`].eventSource = initSSE(account, workspace, baseURI)
        }
      })
    }
  }```